sdv/multi_table/base.py

```diff
@@ -77,7 +77,8 @@ def _initialize_models(self):
         with disable_single_table_logger():
             for table_name, table_metadata in self.metadata.tables.items():
                 synthesizer_parameters = self._table_parameters.get(table_name, {})
-                metadata = Metadata.load_from_dict(table_metadata.to_dict())
+                metadata_dict = {'tables': {table_name: table_metadata.to_dict()}}
+                metadata = Metadata.load_from_dict(metadata_dict)
```

Comment:
was this crashing?

Reply:
No it was causing the single table metadata name to be missing since it gave it a default table (it was in a SingleTableMetadata format)